ThunderCloud/CollectionCell.swift

```diff
@@ -75,19 +74,19 @@ open class CollectionCell: StormTableViewCell {
 	
 	private func sharedInit() {
 		
-		pageControl.currentPage = 0
-		pageControl.pageIndicatorTintColor = .lightGray
-		pageControl.currentPageIndicatorTintColor = ThemeManager.shared.theme.mainColor
-		pageControl.isUserInteractionEnabled = false
+		pageControl?.currentPage = 0
```

Comment:
Could `pageControl` and `collectionView` be unwrapped above, so we don't unwrap them inline for every time it's used? Not sure if it'd bring any performance benefits at all, but it'd save having to do it everywhere :)

Reply:
Done 👍